Woof/Woof/Model/Sitter/Sitter+DummyData.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+extension Sitter {
+    static let dummySitter = Sitter(
+        id: UUID(),
+        name: "Emily",
+        surname: "Doe",
+        phone: "1234567890",
+        avatarUrl: URL(string: "https://link.appcoda.com/testimage"),
```

Comment:
How we can test avatar while we don't have a network connection?
Can we have an URL to resource on a file system?